astropy/coordinates/baseframe.py

```diff
@@ -1354,7 +1355,7 @@ def is_equivalent_frame(self, other):
 
         Parameters
         ----------
-        other : BaseCoordinateFrame
+        other : :class:`~astropy.coordinates.BaseCoordinateFrame`
```

Comment:
Just checked the behavior, and this should actually be
```suggestion
        other : frame-instance
```